### DISCUSSION TITLE
next/image and proportions

### DISCUSSION THREAD

**Discussion by artecoop** [vercel/next.js#18780] [Help]
I don't understand how to use this component and keeping proportions for my image.
Let's say I wanna show images which have coherent proportions that slightly differs in width or height. Assume 800x600 for most of them, and sometimes 760x600, 830x580 etc...

If I use 
```
<Image src={image-url} width={800} height={600} alt="meaningful content" />
```
although is good for the majority of my images, some of them are stretched.

There is a way to keep proportions before rendering?

**Comment by agcty** [comment]
Yes that's exactly what I'm trying to do but it doesn't seem to be possible, check out the discussion here and consider upvoting so the next.js devs see it. https://github.com/vercel/next.js/discussions/18739
